benchmarks/vector_db_compare/rag_qdrant_runner.py

```
@@ -0,0 +1,256 @@
+        deadline = time.monotonic() + 90
+        last = None
+        trigger_client = None
+        while time.monotonic() < deadline:
+            candidate = None
+            try:
+                server_info(self.args.url)
+                candidate = self.factory()
+                candidate.get_collection(self.args.collection)
+                trigger_client = candidate
+                break
+            except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)